openlibrary/macros/LoanStatus.html

```diff
@@ -22,6 +22,8 @@
 $ waiting_loan_total_time = time() - waiting_loan_start_time
 $ my_turn_to_borrow = waiting_loan and waiting_loan['status'] == 'available' and waiting_loan['position'] == 1
 
+$if secondary_action:
+  $:macros.BookPreview(ocaid, linkback=not no_index, for_desktop=False)
```

Comment:
@jimchamp this is excellent guidance, thank you! Makes sense that I shouldn't just stick a brand new call to `BookPreview` in there.  😆 

I have pushed an updated version of this file in my latest commit; could you make sure the edits I made are as you expect them?

One issue is that on my local env, I can now no longer see the 'Preview' link. I'm guessing that's because the sample books I can view are not as they would be in the prod env (i.e. I am guessing nothing passes the `is_lendable` flag).

If I move the `BookPreview` macro outside of the `$elif availability.get('is_lendable'):` conditional, I can see (and preview) books again on my local env, but the placement of the link is sandwiched in between the 'Search Inside' and Status ('Want to Read' / 'Already Read' / etc.) buttons, so I think I might still have it wrong...

Here's how it looks if I move the `BookPreview` macro above the `$elif availability.get('is_lendable'):` conditional ('new' line 82):
<img width="636" alt="Screen Shot 2023-04-26 at 10 15 43 PM" src="https://user-images.githubusercontent.com/13765801/234765652-a8ecc723-c158-4b5a-947e-d3ebb8e7eda2.png">

I can jump on a quick call or Slack huddle to discuss in detail -- just let me know.